src/jit/SimdArm64Inl.h

```diff
@@ -96,6 +106,18 @@ static void simdEmitI64x2Mul(sljit_compiler* compiler, sljit_s32 rd, sljit_s32 r
     simdEmitOp(compiler, SimdOp::umlal | SimdOp::S4, rd, tmpReg2, tmpReg1);
 }
 
+static void simdEmitFloatPMin(sljit_compiler* compiler, sljit_s32 rd, sljit_s32 rn, sljit_s32 rm, SimdOp::FloatSizeType size)
+{
+    simdEmitOp(compiler, SimdOp::fcmgt | size, rd, rn, rm);
+    simdEmitOp(compiler, SimdOp::bsl | SimdOp::B16, rd, rm, rn);
+}
+
+static void simdEmitFloatPMax(sljit_compiler* compiler, sljit_s32 rd, sljit_s32 rn, sljit_s32 rm, SimdOp::FloatSizeType size)
+{
+    simdEmitOp(compiler, SimdOp::fcmgt | size, rd, rm, rn);
+    simdEmitOp(compiler, SimdOp::bsl | SimdOp::B16, rd, rm, rn);
+}
```

Comment:
These two looks like the same. Btw is it for floats not for ints? I think float has special opcodes because if the nans.